nginx/default.conf

```diff
@@ -0,0 +1,14 @@
+
+server {
+    listen       80;
+    server_name  frontend;
+
+    root /src;
+
+    location / {
+        index  index.html index.htm;
```

Comment:
Consider `proxy_pass` to redirect traffics to request processor. : D

Reply:
If we have the backend server for the frontend, we can easily remove Nginx. :D